code/modules/paperwork/ticketmachine.dm

```diff
@@ -102,14 +103,18 @@
 	handle_maptext()
 
 /obj/machinery/ticket_machine/proc/handle_maptext()
-	switch(ticket_number) //This is here to handle maptext offsets so that the numbers align.
-		if(0 to 9)
-			maptext_x = 13
-		if(10 to 99)
-			maptext_x = 10
-		if(100)
-			maptext_x = 8
-	maptext = "<font face='Small Fonts'>[ticket_number]</font>"
+	if(!dispense_enabled)
+		maptext_x = 13
+		maptext = "<font face='Verdana'>X</font>"
+	else
+		switch(ticket_number) //This is here to handle maptext offsets so that the numbers align.
+			if(0 to 9)
+				maptext_x = 13
+			if(10 to 99)
+				maptext_x = 10
+			if(100)
+				maptext_x = 8
+		maptext = "<font face='Small Fonts'>[ticket_number]</font>"
```

Comment:
```suggestion
		maptext = "<font face='Verdana'>X</font>"
		return
	switch(ticket_number) //This is here to handle maptext offsets so that the numbers align.
		if(0 to 9)
			maptext_x = 13
		if(10 to 99)
			maptext_x = 10
		if(100)
			maptext_x = 8
	maptext = "<font face='Small Fonts'>[ticket_number]</font>"
```
Early returns are preferred. Will make it a bit easier to read